### PR TITLE
make the page edge to edge

### PR DESCRIPTION
**Pull request by teolemon**
make the page edge to edge

### CODE REVIEW

File: lang/en/texts/better-food-choices-for-a-better-health-lets-mobilize.html [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<!-- no side column -->
+
+<div class="row">
+	<div class="text-center small-12 medium-4 columns">
```
Comment (stephanegigandet) [inline lang/en/texts/better-food-choices-for-a-better-health-lets-mobilize.html:4]: medium-4 will make the column 4/12 = one third of the screen on medium screens, is that intended? If not just put "small-12".

Comment (teolemon) [inline lang/en/texts/better-food-choices-for-a-better-health-lets-mobilize.html:4]:
```suggestion
	<div class="text-center small-12 small-12 columns">
```

Comment (teolemon) [reply lang/en/texts/better-food-choices-for-a-better-health-lets-mobilize.html]: fixed

File: lang/en/texts/better-food-choices-for-a-better-health-lets-mobilize.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!-- no side column -->
 
 <div class="row">
-	<div class="text-center small-12 medium-4 columns">
+	<div class="text-center small-12 small-12 columns">
```
Comment (stephanegigandet) [inline lang/en/texts/better-food-choices-for-a-better-health-lets-mobilize.html:4]:
```suggestion
	<div class="text-center small-12 columns">
```

**Pull request merged**